bin/blocks-plot

```diff
@@ -131,16 +133,17 @@ if __name__ == "__main__":
     logging.basicConfig()
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("--list", "-l", action="store_true", 
+    parser.add_argument("--list", "-l", action="store_true",
         help="List all channels contained in the experiments and exit")
-    parser.add_argument("--channels", "-c", type=str, 
+    parser.add_argument("--channels", "-c", type=str,
         help="A comma separated list of channels to plot. " + \
```

Comment:
Even though this isn't part of this PR, there's no need for the `+` and the `\` here. Strings are automatically concatenated within parentheses.